lib/hexpm/accounts/user_twofactor.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Hexpm.Accounts.UserTwoFactor do
+  use Hexpm.Web, :schema
+
+  @types ~w(totp disabled)
+
+  embedded_schema do
+    field :enabled, :boolean, default: false
+    field :type, :string, default: "disabled"
```

Comment:
It's supposed to identify whether the 2FA is via TOTP or something else (SMS). Not sure whether it's worth keeping in at this point.